webapp/src/Service/ImportExportService.php

```diff
@@ -844,12 +847,21 @@ public function importAccountsJson(array $data, ?string &$message = null, ?array
                 case 'team':
                     $roles[] = $teamRole;
                     break;
+                case 'staff':
+                    $roles = $clarHandler;
+                    // Also add balloon role;
                 case 'balloon':
                     $roles[] = $balloonRole;
                     break;
```

Comment:
Cds also needs source code reader.
Also if we do this, do we maybe want a special case for WF? Where the type is not cds (since that is not in spec) but the username is

Reply:
Can't we add the CDS as a role to the spec? But I'm fine with an extra if statement that if the type is either cds or the username is.

Reply:
I don’t think we will het that role approved, especially since the permissions are very specific to us (we don’t want them to log in, so why should others care about a special role?$